src/test/java/tests/TyposTest.java

```diff
@@ -0,0 +1,20 @@
+package tests;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TyposTest extends BaseTest {
+
+    @Test(description = "Checking text after refreshing the page")
+    public void checkTextAfterRefresh() {
+        typosPage.openPage();
+        typosPage.isPageOpened();
+
+        while (typosPage.getText().equals(typosPage.wrongText)) {
```

Comment:
Didn't understand the idea of this test. Test should catch bugs and not refreshing the page to wait for the proper result.  Main idea here: test should fail sometimes if page displays incorrect result.

Reply:
You're right. Removed